drivers/net/bnxt/bnxt_ring.c

```diff
@@ -84,6 +85,9 @@ void bnxt_init_ring_grps(struct bnxt *bp)
  * tx bd ring - Only non-zero length if tx_ring_info is not NULL
  * rx bd ring - Only non-zero length if rx_ring_info is not NULL
  */
+//#define ROUNDUP(x) RTE_CACHE_LINE_ROUNDUP(x)
+#define ROUNDUP(size) \
+	(4096 * ((size + 4096 - 1) / 4096))
```

Comment:
It's not clear at this time if the correct value is page size or 4096.  It is apparently most likely to be 4096.